pkgs/development/python-modules/kornia-rs/default.nix

```diff
@@ -30,20 +30,25 @@ buildPythonPackage rec {
 
   buildInputs = lib.optional stdenv.hostPlatform.isDarwin libiconv;
 
-  cargoRoot = "py-kornia";
+  postUnpack = ''
+    cp -v ${./Cargo.lock} source/${cargoRoot}/Cargo.lock
```

Comment:
Submit this file as an upstream PR and apply the diff here so we don't bloat nixpkgs with the lock file?

Reply:
upstream might do it: https://github.com/kornia/kornia-rs/pull/97#issuecomment-3226515989